selfdrive/test/test_car_models.py

```diff
@@ -389,10 +389,18 @@ def get_route_log(route_name):
     'carFingerprint': SUBARU.IMPREZA,
     'enableCamera': True,
   },
+  "95441c38ae8c130e_2020-06-08--12-10-17": {
+    'carFingerprint': SUBARU.FORESTER_PREGLOBAL,
+    'enableCamera': True,
+  },
   "df5ca7660000fba8|2020-06-16--17-37-19": {
     'carFingerprint': SUBARU.LEGACY_PREGLOBAL,
     'enableCamera': True,
   },
+  "5ab784f361e19b78|2020-06-08--16-30-41": {
+    'carFingerprint': SUBARU.OUTBACK_PREGLOBAL,
+    'enableCamera': True,
+  },
```

Comment:
Let's add a comment that these are routes from the dashcam merge. Once we bring these out of dashcam, I'd like to get new routes.

Reply:
ok, done